pytorch_lightning/metrics/functional/classification.py

```diff
@@ -22,9 +22,10 @@
 
 
 def stat_scores(
-        pred: torch.Tensor,
-        target: torch.Tensor,
-        class_index: int, argmax_dim: int = 1,
+    pred: torch.Tensor,
```

Comment:
Could you remove all formatting changes?
I assume that what is actually changing in this file is that you are removing the old accuracy implementation?

Reply:
Sure. These formatting changes appeared bc I automatically formatted it with black 😅